### PR TITLE
[GSK-2807] Documentation polishing

### PR DESCRIPTION
**Pull request by AbSsEnT**
## Description

GSK-2806
GSK-2707
GSK-2767

## Type of Change

- [x] 📚 Examples / docs / tutorials / dependencies update
- [x] 🔧 Bug fix (non-breaking change which fixes an issue)
- [ ] 🥂 Improvement (non-breaking change which improves an existing feature)
- [ ] 🚀 New feature (non-breaking change which adds functionality)
- [ ] 💥 Breaking change (fix or feature that would cause existing functionality to change)
- [ ] 🔐 Security fix

### CODE REVIEW

File: docs/knowledge/llm_vulnerabilities/index.rst [hunk 1]
```diff
@@ -6,168 +6,15 @@ Key vulnerability categories
...
-   injection/index
-   robustness/index
-   formatting/index
-   disclosure/index
-   stereotypes/index
-
-By conducting a `Giskard scan <../../open_source/scan/index.md>`_, you can proactively identify and address these vulnerabilities to ensure the reliability, fairness, and robustness of your LLMs.
-
-
-How does the LLM Scan work?
```
Comment (luca-martial) [inline docs/knowledge/llm_vulnerabilities/index.rst:23]: @AbSsEnT the stuff after this line has to be kept

**Review by luca-martial — CHANGES_REQUESTED** [review]

**Review by luca-martial — CHANGES_REQUESTED** [review]

File: docs/knowledge/llm_vulnerabilities/index.rst (hunk 1 above)
Comment (AbSsEnT) [reply docs/knowledge/llm_vulnerabilities/index.rst:23]: Done

**Review by AbSsEnT — COMMENTED** [review]
@luca-martial the contents was returned to its original place.

**Review by luca-martial — APPROVED** [review]
Ok looks good now - ill let @rabah-khalek review the rest since you addressed 3 different linear cards in 1 PR (better to separate next time)

**Comment by AbSsEnT** [comment]
@luca-martial yes, makes sense to me to separate in the future. However, since there were a lot of small issues dedicated to the dame goal of polishing documentation, I decided to submit everything at once. But yes, separation is good)

**Comment by Hartorn** [comment]
@luca-martial @rabah-khalek Can we merge this, or are we expecting changes ?

**Review by rabah-khalek — APPROVED** [review]

**Pull request merged**